submission/persistence-common-api/src/main/kotlin/ac/uk/ebi/biostd/persistence/common/model/SubmissionRequest.kt

```diff
@@ -15,9 +19,17 @@ data class SubmissionRequest(
     val currentIndex: Int,
     val modificationTime: OffsetDateTime,
     val silentMode: Boolean,
+    val processAll: Boolean,
```

Comment:
I'd rather name the property `singleJob` because, IMO, with `processAll` it's hard to make the connection with the stages without having the right context. It sounds more like there are several submissions and the system will process them all.